web_src/css/base.css

```diff
@@ -1148,6 +1148,11 @@ a.ui.card:hover {
   border-top-color: var(--color-secondary-alpha-50);
 }
 
+.ui.striped.table > tr:nth-child(2n),
+.ui.striped.table > tbody > tr:nth-child(2n) {
```

Comment:
I see these `stripped` classes are removed, so why these styles come?

Reply:
We did not override this yet in Fomantic and the stripes were barely visible as a result on dark theme, that's why I added this style here now to fix other existing cases of striped tables on dark theme.

Reply:
So these are not for this PR, but for other elements, right?

Reply:
Yes, I guess I can go looking for other elements to show a screenshot. It's a related fix.

Reply:
Here, the branches table with this PR applied:

<img width="648" alt="Screenshot 2023-05-10 at 20 37 39" src="https://github.com/go-gitea/gitea/assets/115237/c46f2ee2-bd15-46a7-872a-6d1fe2a95b09">
<img width="651" alt="Screenshot 2023-05-10 at 20 37 46" src="https://github.com/go-gitea/gitea/assets/115237/1c54e872-ceed-4bde-b0fd-81826da2ebda">